e2e-tests/src/test/resources/streamx-mesh.yml

```diff
@@ -0,0 +1,27 @@
+defaultRegistry: ghcr.io/streamx-dev/streamx
+defaultImageTag: latest-jvm
+tenant: "streamx"
+
+processing:
+  relay:
+    image: reference-relay-processing-service
+    incoming:
+      incoming-pages:
+        topic: inboxes/pages
+      # incoming-assets:
+      #   topic: inboxes/assets
+    outgoing:
+      outgoing-pages:
+        topic: outboxes/pages
+      # outgoing-assets:
```

Comment:
for now you can remove assets configuration - it needs to be implemented in other task